package.json

```diff
@@ -15,10 +15,10 @@
     "semver"
   ],
   "scripts": {
-    "test": "npm run build && npm run lint && mocha -r ts-node/register test/index.spec.ts",
+    "test": "npm run build && npm run lint && ./automation/test.js",
     "build": "tsc",
     "lint": "tslint -t stylish -c tslint.json 'src/**/*.ts' 'test/**/*.ts'",
-    "docs": "npm run build && node generateDocs"
+    "docs": "npm run build && ./automation/docs.js"
```

Comment:
This is preexisting, but a bit odd, and doesn't line up with how our other projects work. Let's fix it: if you stop `docs` running `build`, and make `build` run `docs` instead, then `npm run build` will build _everything_, as people will expect, and the `catch-uncommitted-output.sh` script can just be identical to every other project's.

Reply:
Yep this sounds good, thanks for detangling my spaghetti! :smile: